operator/pkg/operator.go

```diff
@@ -181,6 +238,29 @@ func (o *Operator) Start(ctx context.Context) error {
 	}
 }
 
+// Here we query all the batches that have not yet been verified starting from
+// the latest verified batch by the operator. We also get the prior 5 and check if we need to verify them as well
+// This last thing of getting the last 5 is to make sure we have not missed a batch since they are process in parallel
+// and a higher batch number might have been processed first than the lower one.
+// So getting the last five accounts for that case
```

Comment:
I'm not sure the magic 5 solves the issue if they run in parallel. Imagine a given batch, say batch `3` takes longer than the others, then the others threads advance through blocks `4`, `5`, `6`, up to, say, `10`. Then you'll miss `3`, because `10-5=5`.

Reply:
Yes... I guess no number can cover this cases 100%. But given that each batch is produced every 12 seconds, It might be too improbable for 5 or 10 new batches to finish before the oldest ones. Although increasing the number might help, maybe we should came up with a better method or maybe it is just unnecessary. We should discuss more.

Reply:
Updated!